src/dataset-builder/ConceptSetCreator.test.ts

```diff
@@ -1,22 +1,22 @@
 import { screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import { h } from 'react-hyperscript-helpers';
-import { SnapshotBuilderSettings } from 'src/libs/ajax/DataRepo';
 import { renderWithAppContexts as render } from 'src/testing/test-utils';
 
 import { ConceptSetCreator, toConceptSet } from './ConceptSetCreator';
 import { homepageState } from './dataset-builder-types';
 import { dummyDatasetDetails } from './TestConstants';
 
 describe('ConceptSetCreator', () => {
-  const datasetDetails = dummyDatasetDetails('0');
+  const datasetId = '0';
+  const datasetDetails = dummyDatasetDetails(datasetId);
 
   const renderConceptSetCreator = () => {
     const conceptSetUpdater = jest.fn();
     const onStateChange = jest.fn();
     render(
       h(ConceptSetCreator, {
-        snapshotBuilderSettings: datasetDetails.snapshotBuilderSettings as SnapshotBuilderSettings,
+        dataset: datasetDetails,
```

Comment:
Usually I like to declare the local variable with the same name as the property argument so this can be written as:
```suggestion
        dataset,
```

Maybe part of the confusion is that `dummyDatasetDetails` returns `DatasetModel` instead of a "dataset details" object so its name should be `dummyDatasetModel`, so it wouldn't look odd to assign its result to a local variable called `dataset`.